### DISCUSSION TITLE
Redshift Syncer throws an error on install

### DISCUSSION THREAD

**Discussion by bmofkn** [thoughtspot/cs_tools#127] [Q&A] [answered]
We want to try out the newly released (or beta) version of the [Redshift Syncer](https://thoughtspot.github.io/cs_tools/syncer/redshift/).

On our first attempt, we received the following error:
`ERROR    No matching distribution found for sqlalchemy-redshift>=1.4.1 `

<img width="1286" alt="redshift syncer error" src="https://github.com/thoughtspot/cs_tools/assets/119008346/bf677fb0-9319-4577-87c6-f9022118b52f">

does [sqlalchemy](https://pypi.org/project/sqlalchemy-redshift/#history) have a version greater than 0.8.14?  Any ideas on where I can find a version >= 1.4.1?

Thanks!

**Comment by boonhapus** [comment, marked as answer]
You gotta stop livin' in the past @bmofkn 🤪 

I have no idea how I wrote the manifest as 1.4.1 ... I spent a few hours testing this in both a local dev and our production environment. It should definitely be 0.8.14 ! I'll have this fixed before Monday.

**Reply by bmofkn** [reply]
Thanks for [updating the redshift syncer](https://github.com/thoughtspot/cs_tools/releases/tag/v1.5.4) @boonhapus!  I was successfully able to use it via `cs_tools tools searchable metadata --config prod_ts --syncer redshift://rssyncerDefinition.toml` earlier today 🙂
